cli/src/commands/generate/components/property.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+
+export function getStringPlaceholder(name: string): string {
+    return '{{ ' + name.toUpperCase().replaceAll('-', '_') + ' }}';
+}
+
+export function getPropertyValue(keyName: string, detail: any): any {
```

Comment:
Would this be accurate based on what you've written in this function?
```typescript
interface Detail {
    const?: unknown,
    type?: 'string' | 'integer' | 'array',
    $ref?: string
}
```

There is the `types.d.ts` file in the root cli dir which this could maybe be added too.

With this you could also change the if statements to be the following, which is a bit nicer

```typescript
if (detail.const) { ... }
```

Reply:
Interesting. What would happen if there are extra properties?

Reply:
this is where you'd then introduce the `[key: string]: unknown` to represent additional properties